wetterdienst/provider/geosphere/observation/api.py

```diff
@@ -723,10 +451,10 @@ class GeosphereObservationRequest(TimeseriesRequest):
     _endpoint = "https://dataset.api.hub.zamg.ac.at/v1/station/historical/{dataset}/metadata/stations"
```

Comment:
Deprecated endpoint , update to : "https://dataset.api.hub.geosphere.at/v1/station/historical/{dataset}/metadata/stations"

from documentation : https://dataset.api.hub.geosphere.at/v1/docs/user-guide/endpoints.html

Reply:
Updated! Thanks @Hrushikesh-HG, for pointing the change in endpoint.